test/SolidityNameAndTypeResolution.cpp

```diff
@@ -321,6 +323,23 @@ BOOST_AUTO_TEST_CASE(comparison_bitop_precedence)
 	BOOST_CHECK_NO_THROW(parseTextAndResolveNames(text));
 }
 
```

Comment:
I think "NameAndTypeResolution" is not the right place for this test. What about ABIJSON?